crates/nu-command/src/bytes/length.rs

```diff
@@ -66,6 +66,21 @@ impl Command for BytesLen {
             },
         ]
     }
+
+    fn is_const(&self) -> bool {
+        true
+    }
+
+    fn run_const(
+        &self,
+        working_set: &StateWorkingSet,
+        call: &Call,
+        input: PipelineData,
+    ) -> Result<PipelineData, ShellError> {
+        let cell_paths: Vec<CellPath> = call.rest_const(working_set, 1)?;
+        let arg = CellPathOnlyArgs::from(cell_paths);
+        operate(length, arg, input, call.head, &nu_protocol::Signals::EMPTY)
```

Comment:
What about something like this?
```suggestion
        operate(length, arg, input, call.head, working_set.permanent().signals())
```